pgpool2-4.6.yaml

```diff
@@ -63,6 +68,59 @@ subpackages:
       runtime:
         - ${{package.name}}
 
+  - name: ${{package.name}}-compat
+    description: "Compatibility package for Upstream pgpool image"
+    dependencies:
+      provides:
+        - pgpool2-compat=${{package.full-version}}
+      runtime:
+        - postgresql
+        - postgresql-client
+        - postgresql-dev
+    pipeline:
+      - name: "Symlink all pgpool binaries to /opt/pgpool-II/bin"
+        runs: |
+          mkdir -p ${{targets.subpkgdir}}/opt/pgpool-II/bin
+          for bin in \
+            pcp_attach_node pcp_detach_node pcp_health_check_stats pcp_node_count \
+            pcp_node_info pcp_pool_status pcp_proc_count pcp_proc_info pcp_promote_node \
+            pcp_recovery_node pcp_stop_pgpool pcp_watchdog_info \
+            pg_enc pg_md5 pgpool pgpool_setup pgproto watchdog_setup wd_cli \
+          ; do
```

Comment:
We could do this instead:

```
for bin in ${{targets.destdir}}/usr/bin/; do....
```

Reply:
Thanks @EyeCantCU, updated PR accordingly.